OpenRA.Game/Input/Hotkey.cs

```diff
@@ -24,6 +24,9 @@ public bool IsValid()
 		public static bool TryParse(string s, out Hotkey result)
 		{
 			result = Invalid;
+			if (string.IsNullOrWhiteSpace(s))
```

Comment:
.Split is actually the offender below and it will also make ` ` an impossible parse.